packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -304,19 +304,19 @@ contract Deploy is Deployer {
         }
         di.run(dii, dio);
 
-        save("L1CrossDomainMessenger", address(dio.l1CrossDomainMessengerImpl()));
-        save("OptimismMintableERC20Factory", address(dio.optimismMintableERC20FactoryImpl()));
-        save("SystemConfig", address(dio.systemConfigImpl()));
-        save("L1StandardBridge", address(dio.l1StandardBridgeImpl()));
-        save("L1ERC721Bridge", address(dio.l1ERC721BridgeImpl()));
+        save("L1CrossDomainMessengerImpl", address(dio.l1CrossDomainMessengerImpl()));
+        save("OptimismMintableERC20FactoryImpl", address(dio.optimismMintableERC20FactoryImpl()));
+        save("SystemConfigImpl", address(dio.systemConfigImpl()));
+        save("L1StandardBridgeImpl", address(dio.l1StandardBridgeImpl()));
+        save("L1ERC721BridgeImpl", address(dio.l1ERC721BridgeImpl()));
 
         // Fault proofs
-        save("OptimismPortal", address(dio.optimismPortalImpl()));
-        save("OptimismPortal2", address(dio.optimismPortalImpl()));
-        save("DisputeGameFactory", address(dio.disputeGameFactoryImpl()));
-        save("DelayedWETH", address(dio.delayedWETHImpl()));
-        save("PreimageOracle", address(dio.preimageOracleSingleton()));
-        save("Mips", address(dio.mipsSingleton()));
+        save("OptimismPortalImpl", address(dio.optimismPortalImpl()));
```

Comment:
Can this be deleted now that https://github.com/ethereum-optimism/optimism/pull/13489 is merged?
```suggestion
```

Reply:
88d88bd84518581c93e94f53fc7348525dc1efdb